src/tweenable.rs

```diff
@@ -631,6 +672,28 @@ impl<T> Tween<T> {
     pub fn clear_completed_event(&mut self) {
         self.event_data = None;
     }
+
+    /// Enable running a one-shot system upon completion.
+    ///
+    /// If enabled, the tween will run a system via a provided [`SystemId`] when the
+    /// animation completes. This is similar to the [`with_completed()`],
+    /// but uses a system registered by [`register_system()`] instead of a callback.
```

Comment:
What's `register_system()` here?

Reply:
Sorry for the delayed response, been dealing with personal crap.

That is [bevy::ecs::world::World::register_system](https://docs.rs/bevy/latest/bevy/ecs/world/struct.World.html#method.register_system).

Reply:
I meant, if you don't tell rustdoc what `register_system()` is, the link will fail to build because rustdoc will not guess that it's a function on Bevy's `World`.

Reply:
Apologies, I clearly don't know what im doing lol.
Would "[`bevy::ecs::world::World::register_system`]" be acceptable? It seems Rustdoc understands it when I include the full path.

Reply:
Sorry, I just realized a better way to do that. Disregard my last comment.